_data/engine-cli/docker_plugin_disable.yaml

```diff
@@ -1,6 +1,9 @@
 command: docker plugin disable
 short: Disable a plugin
-long: Disable a plugin
+long: |-
+  ables a plugin. The plugin must be installed before it can be disabled,
```

Comment:
Surely this should be `Disables a plugin.` Is there some problem with the format of the input file?